moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -863,6 +863,20 @@ std::vector<OMPLPlannerDescription> MoveItConfigData::getOMPLPlanners() const
   spar_stwo.addParameter("max_failures", "5000", "maximum consecutive failure limit. default: 5000");
   planner_des.push_back(spar_stwo);
 
+  OMPLPlannerDescription aitstar("AITstar", "geometric");
+  aitstar.addParameter(
+      "use_k_nearest", "1",
+      "Set whether to use a k-nearest RGG connection model. If false, AITstar uses an r-disc model. Default: 1");
+  aitstar.addParameter("rewire_factor", "1.5",
+                       "Set the rewire factor of the RGG. Valid values: [1.0:0.01:3.0]. Default: 1.5");
```

Comment:
```suggestion
                       "rewire factor of the RGG. Valid values: [1.0:0.01:3.0]. Default: 1.5");
```

Reply:
From my experience smaller rewire factors result in better performance. Can we set the default to `1.001`?